scripts/tools/nrfconnect/generate_nrfconnect_chip_factory_data.py

```diff
@@ -480,6 +501,10 @@ def base64_str(s): return base64.b64decode(s)
                                     help="Provide a path to the Product Attestation Authority (PAA) certificate to generate the PAI certificate. Without providing it, a testing PAA stored in the Matter repository will be used.")
     optional_arguments.add_argument("--paa_key", type=str,
                                     help="Provide a path to the Product Attestation Authority (PAA) key to generate the PAI certificate. Without providing it, a testing PAA key stored in the Matter repository will be used.")
+    optional_arguments.add_argument("--product_appearance", type=str,
```

Comment:
I think it would be easier to add two arguments instead of forcing a user to compose a JSON and remember the right format. You could then use `choices=PRODUCT_FINISH_ENUM.keys()` to automatically validate the available options.